apps/admin-gui-e2e/src/integration/vo-management-perun_admin.spec.js

```diff
@@ -25,238 +28,245 @@ context('Actions', () => {
       .wait('@getVos')
       // assert that the vo was created
       .get('[data-cy=vo-name-link]')
-        .contains('test-e2e-vo')
-        .should('exist')
+      .contains('test-e2e-vo')
+      .should('exist')
   });
 
   it('test add attribute', () => {
 
     cy.intercept('**/attributesManager/setAttributes/vo').as('setAttributes')
       .get(`[data-cy=${dbVoName}]`)
-        .click()
+      .click()
       .get('[data-cy=attributes]')
-        .click()
+      .click()
       .get('[data-cy=add-attributes]')
-        .click()
+      .click()
       // get attribute Notification default language
-      .get('[data-cy=notification-default-language-value]')
-        .type('en')
+      .get(`[data-cy=${addedAttribute}-value]`)
+      .type('en')
       .get('[data-cy=save-selected-attributes]')
-        .click()
-      .wait('@setAttributes')
+      .click()
+      .intercept('**/attributesManager/getAttributes/vo**').as('getAttributes')
+      .wait('@getAttributes')
       // assert that attribute exists
-      .get('[data-cy=notification-default-language-value]')
-        .should('exist')
+      .get(`[data-cy=${addedAttribute}-value]`)
+      .should('exist')
   });
 
   it('test delete attribute', () => {
 
-    cy.intercept('**/attributesManager/removeAttributes/**').as('removeAttributes')
+    cy.intercept('**/attributesManager/getAttributes/**').as('getAttributes')
       .get(`[data-cy=${dbVoName}]`)
-        .click()
+      .click()
       .get('[data-cy=attributes]')
-        .click()
+      .click()
       // get attribute Link to AUP
-      .get('[data-cy=link-to-aup-checkbox]')
-        .click()
+      .get(`[data-cy=${deteleAttribute}-checkbox]`)
+      .click()
       .get('[data-cy=remove-attributes]')
-        .click()
+      .click()
       .get('[data-cy=delete-attributes]')
-        .click()
-      .wait('@removeAttributes')
+      .click()
+      .wait('@getAttributes')
       // assert that attribute exists
-      .get('[data-cy=link-to-aup-checkbox]')
-        .should('not.exist')
+      .get(`[data-cy=${deteleAttribute}-checkbox]`)
+      .should('not.exist')
   });
 
   it('test add vo member', () => {
 
     cy.intercept('**/membersManager/createMember/u').as('createMember')
       .get(`[data-cy=${dbVoName}]`)
-        .click()
+      .click()
       .get('[data-cy=members]')
-        .click()
+      .click()
       .get('[data-cy=add-members]')
-        .click()
+      .click()
       .get('[data-cy=search-members]')
-        .type(`${dbUser}`)
+      .type(`${dbUser}`)
       .get('[data-cy=search-button]')
-        .click()
+      .click()
       .get(`[data-cy=${dbUser}-checkbox]`)
-        .click()
+      .click()
       .get('[data-cy=add-button]')
-        .click()
+      .click()
       .wait('@createMember')
       // assert that member was created
       .get(`[data-cy=${dbUser}-checkbox]`)
-        .should('exist')
+      .should('exist')
   });
 
   it('test remove vo member', () => {
```

Comment:
On this test with checking the checkbox you should wait for the loading of data to the table before doing some checks